### PR TITLE
Feature EPL-1493: First model of the new Navbar

### PR DESCRIPTION
**Pull request by Sankku**

### CODE REVIEW

File: packages/ComponentLibrary/src/components/Nav/Nav.styles.ts [hunk 1]
```diff
@@ -1,51 +1,28 @@
...
+import { CSSProperties } from 'react';
+import { theme } from '../../theme';
 
-export const NavStyles: CSSProperties = {
+const styles: { [key: string]: CSSProperties } = {
+  NavStyles: {
     display: 'flex',
-    width: '93.5rem',
-    height: '3rem',
+    width: '91rem',
```
Comment (tomyrodeghiero) [inline packages/ComponentLibrary/src/components/Nav/Nav.styles.ts:7]: Is this width of **'91rem'** correct?

File: packages/ComponentLibrary/src/components/Nav/Nav.styles.ts [hunk 2]
```diff
@@ -1,51 +1,28 @@
...
+    width: '22.75rem',
+    padding: '0 0.25rem',
+    borderRadius: '6.25rem',
+    background: `var(--Neutral-0, ${theme.palette.baselineColor.transparentNeutral[5]})`,
+  },
+  RightItems: {
+    display: 'flex',
+    height: '3rem',
+    padding: '0.25rem',
+    width: '14rem',
```
Comment (tomyrodeghiero) [inline packages/ComponentLibrary/src/components/Nav/Nav.styles.ts:21]: Is this constant width correct?

File: packages/ComponentLibrary/src/components/Nav/Nav.styles.ts [hunk 3]
```diff
@@ -1,51 +1,28 @@
...
+    borderRadius: '6.25rem',
+    background: `var(--Neutral-0, ${theme.palette.baselineColor.transparentNeutral[5]})`,
+  },
+  boxStyles: {},
```
Comment (tomyrodeghiero) [inline packages/ComponentLibrary/src/components/Nav/Nav.styles.ts:25]: The **boxStyles** is empty 🧐

File: packages/ComponentLibrary/src/components/Nav/Nav.tsx [hunk 4]
```diff
@@ -1,18 +1,27 @@
...
-import SearchIcon from '../../assets/icons/search.svg';
-import { NavStyles, boxStyles } from './Nav.styles';
-import RigthButtons from './RigthComponents/RightButtons';
-import Logo from './LeftComponents/Logo';
-import { AppIcon } from './LeftComponents/IconApp';
+import React, { ReactNode, useState } from 'react';
+import styles from './Nav.styles';
+import RightButtons from './RigthComponents/RightButtons';
+import SearchInputWithVoice from '../Input/TextInput/TextInputAutocomplete/SearchInputWithVoice';
+interface NavProps {
```
Comment (tomyrodeghiero) [inline packages/ComponentLibrary/src/components/Nav/Nav.tsx:5]: Move **interface** to its corresponding file

**Review by tomyrodeghiero — CHANGES_REQUESTED** [review]

**Review by tomyrodeghiero — APPROVED** [review]

File: packages/ComponentLibrary/src/components/ProfileModal/UserProfile.tsx [hunk 5]
```diff
@@ -1,33 +1,28 @@
...
-  svgContainerStyles,
-  profileDetailsStyles,
-  nameStyles,
-  emailStyles,
-  logoutButtonStyles,
-} from './ProfileModal.styles';
+import { styles } from './ProfileModal.styles';
 import BackgroundGradient from '../../assets/images/backgroundGradient.svg';
 import Logout from '@mui/icons-material/Logout';
 import { user } from './UserMock';
```
Comment (sjcuello) [inline packages/ComponentLibrary/src/components/ProfileModal/UserProfile.tsx:5]: This component is using mocks

**Review by sjcuello — APPROVED** [review]

**Pull request merged**